src/DIRACbenchmark/dirac_benchmark.py

```diff
@@ -0,0 +1,295 @@
+
+""" DIRAC Benchmark 2012 by Ricardo Graciani, and wrapper functions to
+    run multiple copies in parallel by Andrew McNab.
+    This file (DIRACbenchmark.py) is intended to be the ultimate upstream
+    shared by different users of the DIRAC Benchmark 2012 (DB12). The
+    canonical version can be found at https://github.com/DIRACGrid/DB12
+    This script can either be imported or run from the command line:
+    ./DIRACbenchmark.py NUMBER
+    where NUMBER gives the number of benchmark processes to run in parallel.
+    Run  ./DIRACbenchmark.py help  to see more options.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import os
+import sys
+import random
+import multiprocessing
+
+VERSION = "00.04 DB12"
```

Comment:
Is this version used by Pypi?
@fstagni how should we number the version? `0.0.5`? `0.1.0`? `1.0.0`?

Reply:
let's start with `1.0.0`